build/buildall

```diff
@@ -50,7 +57,7 @@ function bloopInstall() {
       mkdir -p "$bloop_config_dir"
       rm -f "$bloop_config_dir"/*
 
-      mvn install ch.epfl.scala:maven-bloop_2.13:1.4.9:bloopInstall -pl dist -am \
+      mvn install ch.epfl.scala:maven-bloop_${BLOOP_SCALA_VERSION}:${BLOOP_VERSION}:bloopInstall -pl dist -am \
```

Comment:
These changes have nothing to do with ExistenceJoin and really should be a separate PR.  Otherwise if we need to revert one we end up reverting them both, and the PR summary "hides" the unrelated changes bundled with it.